CONTRIBUTING.md

```diff
@@ -389,6 +390,17 @@ high value in its `priority` (greater than 100 is recommended) to avoid the erro
 
 Also, we can hide an element from the view using `invisible="1"`.
 
+#### Demo Records
+
+Suffix all demo record XML IDs with `demo`. This allows them to be easily
+distinguished from regular records, which otherwise requires examining the
+source or reinstalling the module with demo data disabled.
+
+```xml
+<record id="not_a_real_user_demo" model="res.users">
```

Comment:
IME, XML IDs generally show up in the context of record creation or assignment to a relational field, both of which make it fairly easy to tell what model is involved. 

That being said, I'm not opposed to this rule, but it's not part of the current guidelines as I understand them. The section on XML ID naming only has subsections on naming for specific models (menus, views, actions, groups, and rules) and on inheritance. Do you want to add this to the PR?